binda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "binda" %}
+{% set version = "1.0.22" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/binda-{{ version }}.tar.gz
+  sha256: 4c902a5cf5034eaf9e88e1ad04e9108121da72dc1c2ea71cd3520a1f3fff7691
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
```

Comment:
Items under `[build-system]` requires should go in the `host` requirements section.